calculator/parser.py

```diff
@@ -0,0 +1,132 @@
+BINARY_OPERATORS = ["+", "-", "*", "/"]
+
+
+def binary_operator(num_1, num_2, op):
```

Comment:
Can we name these more clearly?  For instance, `num_1` and `num_2` doesn't tell me anything about the order of them.  I'd call them something like `lhs` and `rhs` (left/right hand side) to indicate the order.